packages/tools/src/launchEditor.ts

```diff
@@ -164,6 +200,58 @@ function findRootForFile(
   });
 }
 
+// On windows, find the editor executable path even if its not in the users path
+function editorWindowsLaunchPath(editor: string) {
+  try {
+    execSync(`dir "${editor}"`, {stdio: 'ignore'});
+    // Editor is a full path to an exe, we can just launch it
+    return editor;
+  } catch (error) {
+    // ignore
+  }
+
+  try {
+    execSync(`where ${editor}`, {stdio: 'ignore'});
+    // Editor is on the path, we can just launch it
+    return editor;
+  } catch (error) {
+    // ignore
+  }
+
+  try {
+    const editorNames = Object.values(COMMON_WINDOWS_EDITORS);
+    const editorImageNames = Object.keys(COMMON_WINDOWS_EDITORS);
+    for (let i = 0; i < editorNames.length; i++) {
+      const editorName = editorNames[i];
+      if (editor === editorName) {
```

Comment:
I wonder if the case-sensitive comparison is good here. I never can predict what casing is used in Windows. It often seems to be tool-specific.